src/vision/projects/StrUI

```diff
@@ -0,0 +1,48 @@
+# ⚡ Projects: Strui (GUI Framework)
+
+## What is this?
+This is a sample project for use within the various "status quo" or "shiny future" stories.
+
+## Description
+Finally a Rust GUI Framework for the masses :)
+
+It uses Streams as a way of Event Handling:
```

Comment:
Hmm, I'm considering. This feels really different from the other projects in that it is kind of half-way to a user story, and has code examples and the like. I would be curious to get an idea what stories you'd like to write that are tied to this project. 

I do think having a project representing GUIs or other use cases is a good idea.

Reply:
Not sure why it didn't cross my mind, but it makes absolute sense to use a GUI Application as a Project. It's more generic and I'm sure I can bring across the points I wanted to make. Will rewrite tomorrow.

I have quite a few *shiny future* scenarios in mind.

The only current one is:
I'm talking about this Trait, not sure if this is in scope: https://docs.rs/futures/0.3.13/futures/stream/trait.StreamExt.html
If you have some sort of middleware, that works by getting a stream, manipulating it (using ```filter```, ```map``` or similar) and returning that manipulated stream. All well and good, except when you shorten it's lifetime like ```some_stream.take(1)```. All those stream manipulations that return a stream, that indicates it's done sooner than the original one, are most likely the wrong choice in my scenario and can wreck the whole system. Would be nice to make sure a user cannot accidentally use those.